src/Geo/AABB.php

```diff
@@ -58,7 +58,7 @@ public function copy() : AABB
     /**
      * Returns the center of the AABB
      */
-    public function getCenter() : Vec3
+    public function getCenter() : float|Vec3
```

Comment:
Is always `Vec3` phpstan unfortunately is no able to handle the operator overloading properly. 

But Vec3 * float = Vec3